src/Locale/de/cake.po

```diff
@@ -246,7 +246,7 @@ msgstr "Start"
 
 #: View/Widget/DateTimeWidget.php:540
 msgid "January"
-msgstr "Januar"
+msgstr "Jänner"
```

Comment:
According to [this](https://wiki.mozilla.org/L10n:Locale_Codes), there is an extended naming for locales. So I guess the "correct" way of doing this would be to create a locale like ``de_AT_southtirol`` or something like that.

But this is theory only as the software in question would need to support it.